contracts/Smt.metasol

```diff
@@ -20,7 +20,7 @@ library SMT{{hashname}} {
         require(_depth < 255, "Depth must be < 255");
         tree.emptySubtreeCache[0] = {{hash}}("");
         // current depth == 0
-        tree._calculateEmptySubtreeHash(_depth);
+        _calculateEmptySubtreeHash(tree, _depth);
```

Comment:
Why this change? If you're gonna stick with it, should probably remove `using SMT{{hashname}} for MerkleTree` line

Reply:
>Why this change?

if it's private method (`_calculateEmptySubtreeHash` now private) i can not call `tree._calculateEmptySubtreeHash(_depth)` i need to use `_calculateEmptySubtreeHash(tree, _depth)`

>If you're gonna stick with it, should probably remove using `SMT{{hashname}} for MerkleTree` line

i don't get it sorry